demos/python_demos/common/models/model.py

```diff
@@ -0,0 +1,38 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+class Model:
+    def __init__(self, ie, model_path, logger=None):
+        self.logger = logger
```

Comment:
Why are you passing loggers around? Just do `self.logger = logging.getLogger()` and remove all the `if self.logger` constructs.

Reply:
Done